programs/bpf_loader/src/syscalls/cpi.rs

```diff
@@ -9,6 +9,7 @@ use {
         memory_region::{MemoryRegion, MemoryState},
     },
     solana_sdk::{
+        account_info::{ VmAccountInfo, VmBoxOfRefCell, },
```

Comment:
```suggestion
        account_info::{VmAccountInfo, VmBoxOfRefCell},
```
You can run `clippy` and `fmt` to fix lint issues.